src/cli/telemetry/telemetry.ts

```diff
@@ -195,21 +197,31 @@ const CONFIG_PROPS_TO_ANONYMIZE: ReadonlyArray<ConfigStringKeys> = [
   'cacheDir',
   'configPath',
   'tsconfig',
-];
+] satisfies ReadonlyArray<ConfigStringKeys>;
 
 // Props we delete entirely from the config for telemetry
 //
 // TODO(STENCIL-469): Investigate improving anonymization for tsCompilerOptions and devServer
-const CONFIG_PROPS_TO_DELETE: ReadonlyArray<keyof d.Config> = [
+const CONFIG_PROPS_TO_DELETE = [
   'commonjs',
   'devServer',
   'env',
   'logger',
+  'nodeResolve', // this contains regexes which won't serialize to JSON
   'rollupConfig',
   'sys',
   'testing',
   'tsCompilerOptions',
-];
+  'watchIgnoredRegex', // this contains regexes which won't serialize to JSON
+] satisfies ReadonlyArray<keyof d.Config>;
```

Comment:
the new `satisfies` keyword here is super handy - basically what it does is if you do something like

```ts
const Obj satisfies MyType;
```

it will check to ensure that `Obj` _satisfies_ the `MyType` type but it will _not_ change the type of `Obj` to be type `MyType` -- instead, it will keep whatever type the type checker is able to infer for `Obj`. This could be the same type as `MyType`, but it could also be a more specific type that we'd like to keep around.

In this case here that's exactly what we want. We want to check that all of the strings in our `ReadonlyArray` here are keys of `d.Config`, so checking that the value satisfies `ReadonlyArray<keyof d.Config>` gives us that check, but we want to keep the type as `Array<"commonjs" | "devServer"...>` i.e. keep the type as being derived from the actual strings that are here.

Super handy!